Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3
+MAINTAINER SCoRe Lab Community <commuity@scorelab.org>
+
+RUN apt-get update
+
+RUN mkdir -p /home/Bassa
```

Comment:
You can set `/home/Bassa` as the `WORKDIR`, then you can simply say, 

`RUN chmod 755 setup.sh`
`RUN setup.sh`

Refer: https://docs.docker.com/engine/reference/builder/#workdir